src/app/pages/tutorial/tutorial.scss

```diff
@@ -1,20 +1,42 @@
 ion-toolbar {
   --background: transparent;
-  --border-color: transparent;
 }
 
-.slide-title {
-  margin-top: 2.8rem;
+ion-content {
+  --overflow: hidden;
 }
 
 .slide-container {
+	display: grid;
+	grid-template-columns: repeat(4, 100%);
+	grid-template-rows: 1fr;
+
+	height: 100%;
+
+  overflow: scroll;
+	scroll-snap-type: x mandatory;
```

Comment:
Can we add `scroll-snap-stop` you can't accidentally swipe over multiple slides: https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-snap-stop